### DISCUSSION TITLE
how to change globalStyle?

### DISCUSSION THREAD

**Discussion by vaynevayne** [mantinedev/mantine#4583] [Q&A]
such as '--mantine-navbar-width'

below is not work
```jsx
import Global from '@emotion/react'
    <Global
            style={{
              ':root': {
                '--mantine-navbar-width': leftOpened ? 250 : 0,
              },
            }}
          >
            <Image />
          </Global>
```

**Comment by ItaiAxelrad** [comment]
It's the plural `styles`. Also, not sure Global styling is the best choice here. Check out the [AppShell](https://mantine.dev/core/app-shell/) docs for some examples on setting navbar width.

**Reply by vaynevayne** [reply]
Can you give me the exact address of the code, instead of a big, empty document, which is nothing more than that? @ItaiAxelrad
